Executive.js

```diff
@@ -11,14 +11,18 @@ class Executive {
 			this.plan.semesters[new_y].add_course(course, new_x);
 			this.renderArrows();
 		};
+		const t = {
+			id: '19S',
```

Comment:
What is the id for? It seems to just be repeating the information in year/season in a different format.